framework/db/CDbConnection.php

```diff
@@ -432,6 +432,7 @@ protected function createPdoInstance()
 		$pdoClass=$this->pdoClass;
 		if(($driver=$this->getDriverName())!==null)
 		{
+			//Need PDO adaptor for sqlite here ???
```

Comment:
This change and the few below it were part of my attempt to get unit tests working.
As far as I can tell this version of yii doesnt have a pdo adaptor for sqlite which means no tests will be able to access to database I was provided with. 
Some minor changes were also made to make this version of yii refer to the updated class names in the latest version of phpunit